chapter02/kds/ex5.pl

```diff
@@ -0,0 +1,9 @@
+use warnings;
+
+print "Enter a string: \n";
+chomp( $str = <STDIN> );
+print "How many times do you want to display $str?: \n";
+chomp( $repeater = <STDIN> );
+
+print $str x $repeater . "\n";
```

Comment:
*Technically* It does ask for the string to be repeated on separate lines, but I'm only pointing this out because I'm being pedantic and the rest of your work seems perfect.